docs/pages/reference/helm-reference/teleport-kube-agent.mdx

```diff
@@ -645,16 +645,20 @@ You can specify multiple selectors by adding elements to the list.
 |----------|---------------|
 | `string` | `nil`         |
 
-Normally the version of Teleport being used will match the version of the chart being installed. If you install chart version
-10.0.0, you'll be using Teleport 10.0.0.
+Normally the version of Teleport being used will match the version of the chart
+being installed. If you install chart version 10.0.0, you'll be using Teleport
+10.0.0. Upgrading the Helm chart will use the latest version from the repo.
 
-You can optionally override this to use a different published Teleport Docker image tag like `10.2.2` or `11`.
+You can optionally override this to use a different published Teleport Docker
+image tag like `10.1.2` or `11`.
 
-See [this link for information on Community Docker image versions](../../installation.mdx#docker).
+<Admonition type="danger">
+`teleportVersionOverride` MUST NOT be used to control the Teleport version.
```

Comment:
```suggestion
`teleportVersionOverride` is intended for development and MUST NOT be used to control the Teleport version in a typical deployment.
```
Maybe something like this to make it more explicit? In general, though, I'm happy with the language here.